buildSrc/src/main/groovy/MuzzlePlugin.groovy

```diff
@@ -324,7 +324,7 @@ class MuzzlePlugin implements Plugin<Project> {
     def config = instrumentationProject.configurations.create(taskName)
 
     if (!muzzleDirective.coreJdk) {
-      def dep = instrumentationProject.dependencies.create("$versionArtifact.groupId:$versionArtifact.artifactId:$versionArtifact.version") {
```

Comment:
Wait, you probably need those in this file 😄

Reply:
Yikes bad regex :)